Cloud Application Manager/Monitoring/Events Tab Overview.md

```diff
@@ -0,0 +1,40 @@
+{{{
```

Comment:
change the name of this file to `events-tab-overview.md` for consistency.

Reply:
I have changed the name.  Thank you.